appcues/src/main/java/com/appcues/ui/AppcuesActivity.kt

```diff
@@ -142,14 +147,14 @@ internal class AppcuesActivity : AppCompatActivity() {
 
         with(state.stepContainer) {
             // apply backdrop traits
-            backdropTraits.forEach {
+            backdropDecoratingTraits.forEach {
                 with(it) { Backdrop() }
             }
             // create wrapper
             contentWrappingTrait.WrapContent { hasFixedHeight, contentPadding ->
-                Box(
-                    contentAlignment = Alignment.TopCenter
-                ) {
+                Box(contentAlignment = Alignment.TopCenter) {
+                    ApplyUnderlayDecoratingTraits(this)
```

Comment:
key change here, now we know if a decorating trait is underlay or overlay,  then we filter and apply those in the correct place. (both for step and container)